src/app/bucket/bucket.component.html

```diff
@@ -21,30 +21,34 @@
 </mat-card-header>
 <mat-card-content>
 <mat-list dense *ngIf="!showOrderButtons" class="app-mat-list">
-    <mat-list-item *ngFor="let objective of bucket!.objectives">
+    <mat-list-item *ngFor="let displayObjective of displayObjectives()">
     <app-objective
-        [objective]="objective" [unit]="unit"
+        [objective]="displayObjective.objective" [unit]="unit"
+        [cumulativeSum]="displayObjective.cumulativeSum"
+        [bucketAllocationLimit]="bucketAllocationLimit()"
         [unallocatedTime]="unallocatedTime"
         [isEditingEnabled]="isEditingEnabled"
         [isReorderingEnabled]="showOrderButtons"
         [otherBuckets]="otherBuckets"
         (onMoveBucket)="moveObjective($event[0], $event[1], $event[2])"
         (onDelete)="deleteObjective($event)"
-        (onChanged)="onObjectiveChanged($event[0], $event[1])"></app-objective>
+        (onChanged)="onObjectiveChanged($event[0], $event[1])" ></app-objective>
     </mat-list-item>
 </mat-list>
 <mat-list dense *ngIf="showOrderButtons" cdkDropList (cdkDropListDropped)="reorderDrop($event)" class="app-mat-list">
-    <mat-list-item *ngFor="let objective of bucket!.objectives" cdkDrag>
```

Comment:
As discussed elsewhere on this PR, this seems to break drag-and-drop.

For now, it might be better not to change this part of the template at all, and make the `app-objective` work correctly (don't show the cumulative sum) if `cumulativeSum` and `bucketAllocationLimit` are not supplied.